python/lsst/pipe/tasks/postprocess.py

```diff
@@ -573,15 +570,15 @@ def getFunctors(self):
         funcs.update(dict(PostprocessAnalysis._defaultFuncs))
         return funcs
 
-    def getAnalysis(self, parq, funcs=None, filt=None):
+    def getAnalysis(self, parq, funcs=None, band=None):
```

Comment:
Breaking API change...

Reply:
If this is used anywhere outside this file (I'd be shocked though), I think it's better that someone see the breakage immediately, because the parquet multi-index column name changed too.